docker-compose.e2e.yml

```diff
@@ -0,0 +1,21 @@
+version: "2"
+
+services:
+  app:
+    build:
+      dockerfile: Dockerfile.dev
+      context: .
+    image: saltcorn/saltcorn:dev
```

Comment:
I don't think we should have `image` here. It is my understanding that then it will pull the image from docker hub instead of building the image from source. I think you want the tests to run against the image from source